### DISCUSSION TITLE
Ability to adjust axes for the Profile method

### DISCUSSION THREAD

**Discussion by wrosko** [networkit/networkit#1145] [Ideas]
Hi there, we have a graph that is growing and we're trying to profile it. however, when doing the standard profile most things are in the same bucket. There is some structure in the graph, but it is quite dense. I tried figuring out if there is a config value I can set but can't find anything. 

ex:
```python
profile.output("HTML",filename="myreport.",directory="/Downloads")
```
<img width="1092" alt="image" src="https://github.com/networkit/networkit/assets/7385473/7ef80f86-591e-4a41-9daa-9fb1542c0db9">

**Comment by wrosko** [comment]
oops, forgot to ask a question

Is it possible to do this? or would it be a feature request?

**Comment by fabratu** [comment]
Hi,

what is it you want to change, exactly? From your plots, it seems that most nodes have degree zero. It this intended or a bug?

Concerning the config: You can generate (via `create` function) or predefine a config. This controls, which measures are actually computed by the Profiler.

**Comment by wrosko** [comment]
@fabratu hi thanks for getting back to me. 

There are many nodes in my graph that have low measures for the above graphs, but there are nodes that have non-zero values. I'm just wondering how I can request that the profiler use a log-scaled axis?

**Comment by fabratu** [comment]
Unfortunately, this feature (basically any API to adjust the visualization of the plots) is not available. However, this seems like a reasonable feature request.
